umm-spec-lib/src/cmr/umm_spec/migration/distance_units_migration.clj

```diff
@@ -0,0 +1,47 @@
+(ns cmr.umm-spec.migration.distance-units-migration
+  "Contains helper functions for migrating between different versions of UMM vertical coordinate system's
+   distance units"
+  (:require
+   [clojure.string :as string]
+   [cmr.common.util :as util]))
+
+(def altitude-distance-units-mapping
+  "Defines mappings of altitude distance units from v1.9 to v1.10."
+  {"HECTOPASCALS" "HectoPascals"
+   "KILOMETERS" "Kilometers"
+   "MILLIBARS" "Millibars"})
+
+(def depth-distance-units-mapping
+  "Defines mappings of depth distance units from v1.9 to v1.10."
+  {"FATHOMS" "Fathoms"
+   "FEET" "Feet"
+   "HECTOPASCALS" "HectoPascals"
+   "METERS" "Meters"
+   "MILLIBARS" "Millibars"})
+
+(defn migrate-distance-units-to-enum
+  "Migrate distance units from string to enum."
+  [c]
+  (let [old-altitude-distance-units (-> c
```

Comment:
Is there any reason these threads couldn't be a `get-in`? I noticed this in a few places

Reply:
No reason. just a habit. Will change to get-in. Done.